uint16 type
string address
---
bool connected
bool result
